.env.sample

```diff
@@ -1,23 +1,38 @@
+# NODE
 NODE_ENV=
+
+# DATABASE
+DATABASE_PORT=
+MYSQL_DATABASE=
+MYSQL_ROOT_PASSWORD=
+DATABASE_URL=
+
+# API
+API_PORT=
+API_URL=
+SESSION_SECRET
+JWT_SIGNING_KEY=
+JWT_VALIDATING_KEY=
+MOCK_ACCOUNT_PASSWORD
+
+# UI
+UI_PORT=
+
+# AUTHENTICATION
 BASIC_AUTH_KEY=
 BASIC_AUTH_SECRET=
-MULESOFT_API_CIS_URL=
-MULESOFT_API_CIS_KEY=
-MULESOFT_API_CIS_SECRET=
-MULESOFT_API_MDM_EA_URL=
-MULESOFT_API_MDM_EA_KEY=
-MULESOFT_API_MDM_EA_SECRET=
+
+# APIM
```

Comment:
What does the "M" mean? Might be worth having full name here or somewhere?

Reply:
APIM = API Management (This will be the product name)